src/network/serverpackethandler.cpp

```diff
@@ -993,18 +993,14 @@ void Server::handleCommand_Interact(NetworkPacket* pkt)
 		[5] u32 length of the next item (plen)
 		[9] serialized PointedThing
 		[9 + plen] player position information
-		actions:
-		0: start digging (from undersurface) or use
-		1: stop digging (all parameters ignored)
-		2: digging completed
-		3: place block or item (to abovesurface)
-		4: use item
-		5: rightclick air ("activate")
 	*/
-	u8 action;
+
+	InteractAction action;
+	*pkt >> (u8 &)action;
+
 	u16 item_i;
-	*pkt >> action;
 	*pkt >> item_i;
```

Comment:
since the complicated reading of action is no longer necessary I'd prefer it to look like before:
```
	InteractAction action;
	u16 item_i;
	*pkt >> (u8 &) action;
	*pkt >> item_i;
```

Reply:
Agreed. Done.